projects/1.19/assets/random-enchants-2/randomenchants2/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+{
+  "enchantment.randomenchants2.solar_enchant": "日光修补",
+  "enchantment.randomenchants2.solar_enchant.desc" : "放在主副手上的装备在光源附近下会缓慢恢复",
+  "enchantment.randomenchants2.obsidian_buster": "黑曜石克星",
+  "enchantment.randomenchants2.obsidian_buster.desc": "提高黑曜石的挖掘速度",
+  "enchantment.randomenchants2.randomness": "随机掉落",
+  "enchantment.randomenchants2.randomness.desc": "开采任意方块时会掉落随机物品",
+  "enchantment.randomenchants2.magnetic" : "磁性",
+  "enchantment.randomenchants2.magnetic.desc" : "装备会将把开采的方块/掉落物传送到您的背包中",

```

```suggestion
  "enchantment.randomenchants2.magnetic.desc" : "装备会将把开采的方块/掉落物传送到背包中",
```
一般来说，说明性文本不应出现第一/第二人称，出现第二人称时一般不使用`您`
此类问题请查看全文并修改类似错误